scripts/o3de/ExportScripts/export_standalone_monolithic_engine_centric.py

```diff
@@ -0,0 +1,160 @@
+
+args = parser.parse_args(o3de_context.args)
+
+if args.quiet:
+    o3de_logger.setLevel(logging.ERROR)
+
+non_mono_build_path = (o3de_context.engine_path) / 'build' / 'non_mono' if args.non_mono_build_path is None else args.non_mono_build_path
+mono_build_path     = (o3de_context.engine_path) / 'build' / 'mono' if args.mono_build_path is None else args.mono_build_path
+
+selected_platform = args.platform
+
+if not selected_platform:
+    o3de_logger.error("Unable to identify default host platform! Please supply the platform using '--platform'. Options are [pc, linux, mac].")
+    sys.exit(1)
+
+o3de_logger.info(f"Selected Project path: {o3de_context.project_path}")
+o3de_logger.info(f"Selected Engine path : {o3de_context.engine_path}")
+o3de_logger.info(f"Build path for non-monolithic executables: {args.non_mono_build_path}")
+o3de_logger.info(f"Build path for monolithic executables: {args.mono_build_path}")
+
+output_cache_path = args.output_path / 'Cache' / selected_platform
+os.makedirs(output_cache_path, exist_ok=True)
+
+# Build project assets and the engine (non-monolithic)
+if args.build_non_mono_tools:
+    process_command(['cmake', '-S', o3de_context.engine_path, '-B', str(non_mono_build_path), '-DLY_MONOLITHIC_GAME=0', f'-DLY_PROJECTS={o3de_context.project_path}'],
+                    cwd=o3de_context.engine_path)
+    process_command(['cmake', '--build', str(non_mono_build_path), '--target', 'AssetBundler', 'AssetBundlerBatch', 'AssetProcessor', 'AssetProcessorBatch', '--config','profile'],
+                    cwd=o3de_context.engine_path)
+    process_command(['cmake', '--build', str(non_mono_build_path), '--target', f'{project_name}.Assets', '--config', 'profile'],
+                    cwd=o3de_context.engine_path)
+
+# Build monolithic game
+process_command(['cmake', '-S', '.', '-B', str(mono_build_path), '-DLY_MONOLITHIC_GAME=1', '-DALLOW_SETTINGS_REGISTRY_DEVELOPMENT_OVERRIDES=0', f'-DLY_PROJECTS={o3de_context.project_path}'],
+                cwd=o3de_context.engine_path)
+    
+if not args.no_game_launcher:
+    process_command(['cmake', '--build', str(mono_build_path), '--target', f'{project_name}.GameLauncher', '--config', args.config],
+                    cwd=o3de_context.engine_path) 
+
+if not args.no_server_launcher:
+    process_command(['cmake', '--build', str(mono_build_path), '--target', f'{project_name}.ServerLauncher', '--config', args.config],
+                    cwd=o3de_context.engine_path)
+
+if not args.no_unified_launcher:
+    process_command(['cmake', '--build', str(mono_build_path), '--target', f'{project_name}.UnifiedLauncher', '--config', args.config],
+                    cwd=o3de_context.engine_path)
+
+# Before bundling content, make sure that the necessary executables exist
+asset_bundler_batch_path = non_mono_build_path / 'bin' / 'profile' / ('AssetBundlerBatch' + ('.exe' if host_asset_platform=='pc' else ''))
```

Comment:
`host_asset_platform` is not the right value to check the executable extension. 
```suggestion
asset_bundler_batch_path = non_mono_build_path / 'bin' / 'profile' / ('AssetBundlerBatch' + ('.exe' if sys.platform=='windows' else ''))
```